source/includes/Patient/patient_identifier_type.md

```diff
@@ -64,27 +128,30 @@ POST /patientidentifiertype
 
 Parameter | Type | Description
 --- | --- | ---
-*name* | string | label for the identifier
+*name* | string | label for the identifier (Required)
 *description* | string | a small description about the patientIdentifier
 *format* | string | a regular expression defining what the identifier text should contain
-*formatDescription* | string | an optional description of the regular expression
-*required* | boolean | a true/false whether every patient MUST have this type
-*checkDigit* | boolean | a true/false whether this identifier has a checkdigit at the end
-*validator* | string | `org.openmrs.patient.IdentifierValidator`
-*locationBehavior* | "REQUIRED" or "NOT USED" | behavior of the location with respect to the identifier 
+*formatDescription* | string | an optional description of the regular expression,(used to explain the requirements of the regular expression in terms a user would understand). For example, a regular expression like `\d{4,8}` could have a description like "Must be a number between 4 and 8 digits in length."
+*required* | boolean | a true/false whether every patient must have this type
+*checkDigit* | boolean | a true/false whether this identifier has a checkdigit at the end (Required)
+*validator* | string | full class name of an [IdentifierValidator](https://docs.openmrs.org/doc/org/openmrs/patient/IdentifierValidator.html) (e.g., [`org.openmrs.patient.impl.LuhnIdentifierValidator`](https://docs.openmrs.org/doc/org/openmrs/patient/impl/LuhnIdentifierValidator.html))
+
+*locationBehavior* | "REQUIRED" or "NOT USED" | behavior of the location with respect to the identifier,"REQUIRED" if a location must be associated with the identifier; "NOT_USED" if the identifier does require a location
+ 
 *uniquenessBehavior* | string | specify the uniqueness of the behaviour, it can be either Unique, Non Unique or Location.
 
 
 ## Update a patientIdentifierType
 
-```console
+```shell
 POST /patientidentifertype/:target_patientidentifiertype_uuid
 {
     "name": "Amani Identifier",
     "description": "Medical record number for Amani Health System",
     "format": "\\d{1,10}-\\d",
     "formatDescription": "Up to ten digts followed by a hyphen and another digit",
     "required": false,
+    "*checkDigit":true,
     "validator": "org.openmrs.patient.impl.LuhnIdentifierValidator",
     "locationBehavior": "NOT_USED",
     "uniquenessBehavior": "UNIQUE"
```

Comment:
@Akayeshmantha I am getting a 500 Internal server error with this payload could you get whats wrong with this Json payload? thanks !!